core/src/main/resources/default-main.conf

```diff
@@ -13,8 +14,13 @@
 # remote.nossl.hosts = *.redhat.com, 10.0.*
 ######################################################################
 
-# remote.nossl.hosts = "10.*,172.*,localhost"
+# Regexes of allowed NON-SSL remote hosts separated by ","
+remote.nossl.hosts = localhost,.apache.org
 
+# Internal Store Validation enabled
```

Comment:
We can remove this, now that we're controlling it from a different config file / class.